exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/RexVisitorComplexExprSplitter.java

```diff
@@ -85,21 +83,21 @@ public RexNode visitCorrelVariable(RexCorrelVariable correlVariable) {
 
   @Override
   public RexNode visitCall(RexCall call) {
-
-    String functionName = call.getOperator().getName();
-
-    List<RexNode> newOps = new ArrayList<>();
+    final List<RexNode> newOps = new ArrayList<>();
```

Comment:
This list population may be replaced with collect.

Reply:
I'd like to leave it as is because of debugging reasons.